src/viur/core/bones/base.py

```diff
@@ -132,12 +133,12 @@ class MultipleConstraints:  # Used to define constraints on multiple bones
     The MultipleConstraints class is used to define constraints on multiple bones, such as the minimum
     and maximum number of entries allowed and whether duplicate values are allowed.
     """
-    minAmount: int = 0  # Lower bound of how many entries can be submitted
+    min: int = 0  # Lower bound of how many entries can be submitted
     """An integer representing the lower bound of how many entries can be submitted (default: 0)."""
-    maxAmount: int = 0  # Upper bound of how many entries can be submitted
+    max: int = 0  # Upper bound of how many entries can be submitted
     """An integer representing the upper bound of how many entries can be submitted (default: 0)."""
-    preventDuplicates: bool = False  # Prevent the same value of being used twice
-    """A boolean value indicating if the same value can be used twice (default: False)."""
+    duplicates: bool = True  # Prevent the same value of being used twice
```

Comment:
According to _principle of least privilege_, i think forbiding duplicates as longs as you not explicitly allow it, would not that bad. I currently see any use-case to allow duplicates, so I would prefer to reject this as long you really need this and enable it.


Following the _principle of least privilege_, I think disallowing duplicates as long as you do not explicitly allow them would not be so bad. I currently see no use case for allowing duplicates, so I would prefer to deny it as long as you really need it and enable it.